src/Parser/EnhancementShaman/CombatLogParser.js

```diff
@@ -0,0 +1,336 @@
+          <dfn data-tip="">
+            Equipped Legendary
+          </dfn>
+        ),
+      });
+    }
+  });
+
+  //add trinkets to item list
+  function getItemInfo(id) {
+    let value;
+    Object.keys(ITEMS).some(function(k) {
+      if(ITEMS[k].id === id) {
+        value = ITEMS[k];
+      }
+    });
+
+    return value;
+  }
+
+    if (nonDpsTimePercentage > 0.3) {
+      results.addIssue({
+        issue: `[NYI] Your non DPS time can be improved. Try to cast damaging spells more regularly (${Math.round(nonDpsTimePercentage * 100)}% non DPS time).`,
+        icon: 'petbattle_health-down',
+        importance: getIssueImportance(nonDpsTimePercentage, 0.4, 0.45, true),
+      });
+    }
+    if (deadTimePercentage > 0.2) {
+      results.addIssue({
+        issue: `Your dead GCD time can be improved. (${Math.round(deadTimePercentage * 100)}% dead GCD time).`,
+        icon: 'spell_mage_altertime',
+        importance: getIssueImportance(deadTimePercentage, 0.35, 0.4, true),
+      });
+    }
+
+    if(flametongueUptime < .85) {
+		results.addIssue({
+        issue: `Your Flametongue uptime of ${formatPercentage(flametongueUptime)}% is below 85%`,
+        icon: SPELLS.FLAMETONGUE_BUFF.icon,
+        importance: getIssueImportance(flametongueUptime, 0.35, 0.4, true),
+      });
+	}
+
+	if(hasHailstorm && frostbrandUptime < .85) {
+		results.addIssue({
+        issue: `Your Frostbrand uptime of ${formatPercentage(frostbrandUptime)}% is below 85%`,
+        icon: SPELLS.FROSTBRAND.icon,
+        importance: getIssueImportance(frostbrandUptime, 0.35, 0.4, true),
+      });
+	} else if(!hasHailstorm && frostbrandUptime > 0) {
+    results.addIssue({
+        issue: `Casting Frostbrand without Hailstorm talent is not recommended`,
+        icon: SPELLS.FROSTBRAND.icon,
+        importance: ISSUE_IMPORTANCE.MAJOR,
+      });
+    }
+
+    if(hasLandslide && landslideUptime < .85) {
+      results.addIssue({
+          issue: `Your Landslide uptime from Rockbiter of ${formatPercentage(landslideUptime)}% is below 85%`,
+          icon: SPELLS.LANDSLIDE_BUFF.icon,
+          importance: getIssueImportance(landslideUptime, 0.35, 0.4, true),
+        });
+    }
+
+    if(hasFuryOfAir && furyofairUptime < .95) {
+      results.addIssue({
+          issue: `Your Fury of Air ${formatPercentage(furyofairUptime)}% uptime is below 95%`,
+          icon: SPELLS.FURY_OF_AIR.icon,
+          importance: getIssueImportance(furyofairUptime, 0.35, 0.4, true),
+        });
+    }
+
+    const castEfficiencyCategories = SPELL_CATEGORY;
+    const castEfficiency = getCastEfficiency(CPM_ABILITIES, this);
+    castEfficiency.forEach((cpm) => {
+        if (cpm.canBeImproved && !cpm.ability.noSuggestion) {
+            results.addIssue({
+                issue: <span>Try to cast <SpellLink id={cpm.ability.spell.id} /> more often ({cpm.casts}/{cpm.maxCasts} casts: {Math.round(cpm.castEfficiency * 100)}% cast efficiency). {cpm.ability.extraSuggestion || ''}</span>,
```

Comment:
See https://github.com/MartijnHols/WoWAnalyzer/blob/de098232ed086189a1ae2a934e3b21439e1f2cf2/src/Parser/HolyPaladin/CombatLogParser.js#L385-L392 for an example of how to write this with the new `stat` property.